### PR TITLE
Further manual fixes

### PR DESCRIPTION
**Pull request by kroening**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: doc/cprover-manual/properties.md [hunk 1]
```diff
@@ -203,20 +203,22 @@ goto-instrument error_example.goto error_example_replaced.goto \
...
+
+error_example.c function internal_error
+[internal_error.assertion.1] line 5 assertion false: FAILURE
+
+** 2 of 2 failed (2 iterations)
+VERIFICATION FAILED
+```
+
+Without the instrumentation step we would have seen
+"verification successful".
```
Comment (tautschnig) [inline doc/cprover-manual/properties.md]: I think this should match the output precisely, i.e., be all-caps.

**Review by tautschnig — APPROVED** [review]

File: doc/cprover-manual/properties.md (hunk 1 above)
Comment (kroening) [reply doc/cprover-manual/properties.md]: done

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 848186b7d914ecaeb191586f3bb1de3513c61a83).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105543007
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 7a0968c6e441b52a58a6ce62986e3009ce42f521).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105543322
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 8b3035f0f74b151bd13857a72fb487744675ba27).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105543693

**Pull request merged**